addons/html_builder/static/src/plugins/background_option/background_option_plugin.js

```diff
@@ -26,6 +26,8 @@ class BackgroundOptionPlugin extends Plugin {
         }
     }
     markColorLevel(editingEl) {
+        // TODO: handle the !(with_colors and with_color_combinations) -> not
+        // o_colored_level. + handle the remove of getSelectorParams
```

Comment:
TODO: also handle the remove of the extends (Parallax (do not know why it extends), WebsiteBackgroundOption)